ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -398,81 +394,10 @@ describe('ExpandedApp', () => {
         });
 
         const launchButton = wrapper.find({
-          'data-test-id': 'RStudio-launch-button',
+          'data-test-id': 'open-RStudio-button',
         });
         expect(launchButton.prop('disabled')).toBeTruthy();
       }
     );
   });
-
-  const createEnabledStatuses = [AppStatus.DELETED, null, undefined];
-  const createDisabledStatuses = minus(
-    ALL_GKE_APP_STATUSES,
-    createEnabledStatuses
-  );
-
-  describe('should allow creating an RStudio app for certain app statuses', () => {
-    test.each(createEnabledStatuses)('Status %s', async (appStatus) => {
-      const wrapper = await component(UIAppType.RSTUDIO, {
-        appName: 'my-app',
-        googleProject,
-        status: appStatus,
-      });
-      appsStub.createApp = jest.fn(() => Promise.resolve({}));
-
-      const createButton = () =>
-        wrapper.find({
-          'data-test-id': `RStudio-create-button`,
-        });
-      expect(createButton().exists()).toBeTruthy();
-      expect(createButton().prop('disabled')).toBeFalsy();
-      expect(createButton().prop('buttonText')).toEqual('Create');
-
-      createButton().simulate('click');
-      await waitOneTickAndUpdate(wrapper);
-
-      expect(appsStub.createApp).toHaveBeenCalledWith(
-        workspace.namespace,
-        defaultRStudioConfig
-      );
-      expect(createButton().prop('buttonText')).toEqual('Creating');
-      expect(createButton().prop('disabled')).toBeTruthy();
-    });
-  });
-
-  describe('should disable the RStudio create button for all other app statuses', () => {
-    test.each(createDisabledStatuses)('Status %s', async (appStatus) => {
-      const wrapper = await component(UIAppType.RSTUDIO, {
-        appName: 'my-app',
-        googleProject,
-        status: appStatus,
-      });
-
-      const createButton = wrapper.find({
-        'data-test-id': `RStudio-create-button`,
-      });
-      expect(createButton.exists()).toBeTruthy();
-      expect(createButton.prop('disabled')).toBeTruthy();
-    });
-  });
-
-  it('should show an error if the initial request to create RStudio fails', async () => {
```

Comment:
The config page has different logic for error handling. I attempted to add test coverage for it, but ran into several issues (I can go into details if needed) and stopped after ~20 minutes.